packages/cli/src/commands/login/index.ts

```diff
@@ -67,20 +68,34 @@ export default async function login(client: Client): Promise<number> {
     return result;
   }
 
+  const isNewLogin = !client.authConfig.token;
+
+  // Save the user's authentication token to the configuration file.
+  client.authConfig.token = result.token;
+
   // If the token was upgraded (not a new login), then don't modify
   // the current scope.
```

Comment:
Per this comment: https://github.com/vercel/vercel/pull/10535/files/54434b23934e9053d56b11ae7f31c1feee1128c7#r1330176372

```suggestion
  // If we have a new login, update `currentTeam`
```